src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildWithLibraryTests.cs

```diff
@@ -75,6 +75,14 @@ public void DotNetBuildLibrary (bool isRelease, bool duplicateAar, bool useDesig
 			libC.OtherBuildItems.Add (new AndroidItem.AndroidAsset ("Assets\\bar\\bar.txt") {
 				BinaryContent = () => Array.Empty<byte> (),
 			});
+			libC.OtherBuildItems.Add (new AndroidItem.None ("AndroidManifest.xml") {
```

Comment:
It errors with:
```
BuildWithLibraryTests.cs(78,46): error CS0426: The type name 'None' does not exist in the type 'AndroidItem'
```

So maybe we do:
```suggestion
			libC.OtherBuildItems.Add (new BuildItem ("None", "AndroidManifest.xml") {
```

Reply:
🤦